src/main/kotlin/dev/minn/jda/ktx/InlineShardManagerBuilder.kt

```diff
@@ -0,0 +1,542 @@
+            field = value
+        }
+    
+    var relativeRateLimit: Boolean = true
+        set(value) {
+            builder.setRelativeRateLimit(value)
+            field = value
+        }
+    
+    var enableCache: Collection<CacheFlag> = emptySet()
+        set(value) {
+            builder.enableCache(value)
+            field = value
+        }
+    
+    var disableCache: Collection<CacheFlag> = emptySet()
+        set(value) {
+            builder.disableCache(value)
+            field = value
+        }
+    
+    var memberCachePolicy: MemberCachePolicy? = null
+        set(value) {
+            builder.setMemberCachePolicy(value)
+            field = value
+        }
+    
+    var sessionController: SessionController? = null
+        set(value) {
+            builder.setSessionController(value)
+            field = value
+        }
+    
+    var voiceDispatchInterceptor: VoiceDispatchInterceptor? = null
+        set(value) {
+            builder.setVoiceDispatchInterceptor(value)
+            field = value
+        }
+    
+    var contextMap: IntFunction<out ConcurrentMap<String, String>>? = null
+        set(value) {
+            builder.setContextMap(value)
+            field = value
+        }
+    
+    var context: Boolean = true
+        set(value) {
+            builder.setContextEnabled(value)
+            field = value
+        }
+    
+    var compression: Compression? = null
+        set(value) {
+            if (value != null) {
+                builder.setCompression(value)
+            }
+            field = value
+        }
+    
+    val eventListeners: MutableCollection<Any> = DelegatingCollection(
+            adder = { item -> builder.addEventListeners(item) },
+            remover = { item -> builder.removeEventListeners(item) },
+                                                                     )
+    
+    val eventListenerProvider: MutableCollection<IntFunction<Any>> = DelegatingCollection(
+            adder = { item -> builder.addEventListenerProvider(item) },
+            remover = { item -> builder.removeEventListenerProvider(item) },
+                                                                                         )
+    
+    var audioSendFactory: IAudioSendFactory? = null
+        set(value) {
+            builder.setAudioSendFactory(value)
+            field = value
+        }
+    
+    var autoReconnect: Boolean = true
+        set(value) {
+            builder.setAutoReconnect(value)
+            field = value
+        }
+    
+    var bulkDeleteSplitting: Boolean = true
+        set(value) {
+            builder.setBulkDeleteSplittingEnabled(value)
+            field = value
+        }
+    
+    var enableShutdownHook: Boolean = true
+        set(value) {
+            builder.setEnableShutdownHook(value)
+            field = value
+        }
+    
+    @Deprecated("Use eventManagerProvider instead")
+    var eventManager: IEventManager? = null
```

Comment:
Remove things that are deprecated

Reply:
It is deprecated, but still supported as a feature. Therefore, I left it in.